jdaviz/configs/cubeviz/plugins/tools.py

```diff
@@ -6,14 +6,26 @@
 from glue.core.roi import RectangularROI
 from glue.core.edit_subset_mode import NewMode
 
+from jdaviz.configs.imviz.plugins.tools import _MatchedZoomMixin
 from jdaviz.core.events import SliceToolStateMessage
+from jdaviz.core.tools import PanZoom
 
 __all__ = []
 
-
 ICON_DIR = os.path.join(os.path.dirname(__file__), '..', '..', '..', 'data', 'icons')
 
 
+@viewer_tool
+class MatchPanZoom2(_MatchedZoomMixin, PanZoom):
+    """Like MatchPanZoom in Imviz but without complicated
+    WCS linking logic because it is not needed for Cubeviz.
+    """
+    icon = os.path.join(ICON_DIR, 'panzoom_match.svg')
+    tool_id = 'jdaviz:panzoommatch2'
+    action_text = 'Pan, matching between viewers'
+    tool_tip = 'Pan (click-drag) and Zoom (scroll) in this viewer to see the same regions in other viewers'  # noqa
```

Comment:
is there any way we can generalize the imviz tool to detect it doesn't need the WCS logic and skip it?  (and then probably move both that and `_MatchedZoomMixin` from imviz/tools to core/tools)

Reply:
```python
        x, y, _ = self.viewer._get_real_xy(image, x, y)
        self.viewer.center_on((x, y))
```

We could spoof `_get_real_xy` as no-op for Cubeviz, yes. But the `center_on` is more controversial. I am not quite ready to introduce Astrowidgets for 3D image cubes.

Reply:
We could also make `on_click` a no-op for Cubeviz but then the tooltip would be wrong for Cubeviz.

Reply:
what about just wrapping that block in `if hasattr(self.viewer, '_get_real_xy')`?

Reply:
Maybe. I'll ponder tomorrow.

Reply:
If I go with your suggestion, all the Cubeviz image viewers will register for clicks that they will never use. Is it really worth it?

https://github.com/spacetelescope/jdaviz/blob/d70425cfeb6bc6b65c3a694abfb719aa4be06568/jdaviz/configs/imviz/plugins/tools.py#L75

Reply:
I guess I should rename it... I was trying to be lazy but you caught me!

... Unless it is considered public API already?

Reply:
I don't think we expose the API for the tool classes themselves.  Do we document how to access tools programmatically (from `viewer.toolbar`, etc) anywhere?

Reply:
Only in `docs/dev/ui_description.rst` but that only uses the string name for the tool, not the class name.